internal/sdkv2provider/resource_cloudflare_dlp_profile.go

```diff
@@ -161,6 +163,8 @@ func resourceCloudflareDLPProfileUpdate(ctx context.Context, d *schema.ResourceD
 		Type: d.Get("type").(string),
 	}
 	updatedDLPProfile.Description, _ = d.Get("description").(string)
+	updatedDLPProfile.AllowedMatchCount, _ = d.Get("allowed_match_count").(uint)
```

Comment:
similar to https://github.com/cloudflare/terraform-provider-cloudflare/pull/2210/files#r1093789250